contents/handbook/growth/sales/sales-operations.md

```diff
@@ -104,8 +104,7 @@ Yes! See [migration tools repo](https://github.com/PostHog/posthog-migration-too
 
 _Can a customer transfer from Cloud to Self-hosted?_
 
-Yes! See [the Replicator app](/docs/apps/replicator) for events and the [migrate meta](https://github.com/PostHog/posthog-migrate-meta) repo for everything else.
-
+Yes! [Raise a support ticket in the app](https://app.posthog.com/home#supportModal) under Data Management.
```

Comment:
I assume this is pretty rare (never?)? Otherwise it's a shame to add more manual support load. :P

Reply:
I know I wasn't sure what to do about this one tbh.